service/src/main/java/org/cbioportal/genome_nexus/service/internal/CancerHotspotServiceImpl.java

```diff
@@ -156,25 +159,29 @@ public List<Hotspot> getHotspotAnnotationsByGenomicLocation(String genomicLocati
         throws VariantAnnotationNotFoundException, VariantAnnotationWebServiceException,
         CancerHotspotsWebServiceException
     {
-        GenomicLocation location = this.notationConverter.parseGenomicLocation(genomicLocation);
-
-        return this.getHotspotAnnotationsByVariant(this.notationConverter.genomicToHgvs(location));
+        VariantAnnotation variantAnnotation = genomicLocationAnnotationService.getAnnotation(genomicLocation);
```

Comment:
Nice 👍 

This is basically doing:
```java
    VariantAnnotation variantAnnotation = genomicLocationAnnotationService.getAnnotation(genomicLocation);
```
instead of
```java
    VariantAnnotation variantAnnotation = variantAnnotationService.getAnnotation(this.notationConverter.genomicToHgvs(this.notationConverter.parseGenomicLocation(genomicLocation)))
```

I think the remaining question is: do we know if these can be expected to return (mostly) the same results? How can we test that?

Reply:
I think it depends what version of vep cache you have locally vs what version of data we get via `vep.url` endpoint. The first version of the call can be configured to fetch data whether locally or from the ensembl endpoint. The second only from the ensembl endpoint.